sidekick/tests/test_views.py

```diff
@@ -1079,3 +1079,106 @@ def test_get_rapidpro_contact_not_in_org(self):
 
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
         self.assertEqual(response.json(), {"error": "Organization not found"})
+
+
+class TurnContextContactFieldsViewTests(APITestCase):
+    def setUp(self):
+        self.api_client = APIClient()
+        self.user = User.objects.create_user(
+            "testuser", "testuser@example.com", "password"
+        )
+        token = Token.objects.get(user=self.user)
+        self.api_client.credentials(HTTP_AUTHORIZATION="Token " + token.key)
+        self.org = Organization.objects.create(id=1, name="Test Org")
+        self.url = reverse("turn-context-contact-fields", args=[self.org.id])
+
+    def test_handshake_response(self):
+        response = self.api_client.post(self.url, {"handshake": True}, format="json")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertIn("version", response.data)
+        self.assertIn("capabilities", response.data)
+        self.assertIn("context_objects", response.data["capabilities"])
+
+    def test_organization_does_not_exist(self):
+        url = reverse("turn-context-contact-fields", args=[999])
+        with patch("sidekick.views.Organization.objects.get") as mock_get:
+            mock_get.side_effect = Organization.DoesNotExist
+            response = self.api_client.post(url, {"handshake": True}, format="json")
+            self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    @patch("sidekick.views.Organization.objects.get")
+    @patch("sidekick.views.TurnContextContactFieldsSerializer")
```

Comment:
Do we need to mock these two?

We created a test org so the get should work and not sure why we would mock a serializer?